src/julia-config-p.h.in

```diff
@@ -0,0 +1,9 @@
+//
+
+#ifndef JULIA_CONFIG_P_H
+#define JULIA_CONFIG_P_H
+
+#define JL_SYSTEM_IMAGE_PATH "@JL_SYSTEM_IMAGE_PATH@"
```

Comment:
I don't think this approach is going to work, since this may need to be changed between the build tree and the installed copy if the relative layouts are different.